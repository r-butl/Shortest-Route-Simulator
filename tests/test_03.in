4 4 75 75
0 3
0 0
1 0
2 1
3 0
0 1 50
0 2 45
2 1 25
1 3 50
