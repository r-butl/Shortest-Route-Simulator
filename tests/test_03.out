Verify Path: 1
120: 0 2 3 
